templates/Controllers/production.html

```diff
@@ -11,11 +11,11 @@ <h1>Production</h1>
 			<span ng-if="!tab.item">
 				<span class="fas fa-question"></span>
 			</span>
-			<img ng-if="tab.item" ng-src="/assets/images/items/{{tab.item.slug}}_64.png" width="32" height="32" title="{{tab.item.name}}" tooltip>
+			<img ng-if="tab.item" ng-src="/assets/images/items/{{tab.item.slug}}_64.png" width="32" height="32" uib-tooltip="{{tab.item.name}}">
```

Comment:
can this be replaced with item icon directive maybe?

Reply:
What do you mean? Which icon?